client/asset/btc/btc.go

```diff
@@ -300,16 +311,10 @@ type ExchangeWallet struct {
 	fallbackFeeRate   uint64
 	useSplitTx        bool
 
-	// In the future, the client may wish to specify minimum confirmations for
-	// utxos to fund orders, and allowing change outputs from DEX-related swap
-	// transaction may be part of this client policy.
-	changeMtx   sync.RWMutex
-	tradeChange map[string]time.Time // TODO: delete fully confirmed change
-
 	// Coins returned by Fund are cached for quick reference and for cleanup on
 	// shutdown.
 	fundingMtx   sync.RWMutex
-	fundingCoins map[string]*compositeUTXO
+	fundingCoins map[wire.OutPoint]*compositeUTXO
```

Comment:
My reservation about this was the pointer in the wire.Outpoint.  Comparing these just compares the pointers, not the hash it points to, so you can't retrieve an something based on knowledge of the hash and vout.

Reply:
Oh yeah. Forgot that was a pointer.